lib/src/components/NxTable/__tests__/NxTable.test.tsx

```diff
@@ -55,25 +56,30 @@ describe('NxTable', function() {
       </NxTable>
     );
 
-    expect(component.find('thead th').length).toBe(2);
-    expect(component.find('thead td')).not.toExist();
-    expect(component.find('tbody td').length).toBe(2);
-    expect(component.find('tbody th')).not.toExist();
+    const table = screen.getByRole('table');
+    const rowGroups = within(table).getAllByRole('rowgroup');
+    const headerRows = within(rowGroups[0]).getAllByRole('row');
+    const headerColumnHeaders = within(headerRows[0]).getAllByRole('columnheader');
+    const headerCells = within(headerRows[0]).queryAllByRole('cell');
+    const bodyRows = within(rowGroups[1]).getAllByRole('row');
+    const bodyColumnHeaders = within(bodyRows[0]).queryAllByRole('columnheader');
+    const bodyCells = within(bodyRows[0]).getAllByRole('cell');
 
-    expect(component.find('thead tr')).toHaveClassName('nx-table-row--header');
-    expect(component.find('thead tr')).toHaveClassName('nx-table-row');
+    expect(table).toBeInTheDocument();
+    expect(rowGroups.length).toBe(2);
+    expect(headerRows.length).toBe(1);
+    expect(headerColumnHeaders.length).toBe(2);
+    expect(headerCells.length).toBe(0);
+    expect(bodyRows.length).toBe(1);
+    expect(bodyColumnHeaders.length).toBe(0);
+    expect(bodyCells.length).toBe(2);
 
-    expect(component.find('th.nx-cell.nx-cell--header').length).toBe(2);
-
-    expect(component.find('tbody tr')).not.toHaveClassName('nx-table-row--header');
-    expect(component.find('td.nx-cell--header')).not.toExist();
-
-    // colSpan should not be getting set on table cells
-    expect(component.find('td').findWhere(td => td.prop('colSpan') != null)).not.toExist();
+    expect(bodyCells[0]).not.toHaveAttribute('colSpan');
+    expect(bodyCells[1]).not.toHaveAttribute('colSpan');
   });
 
   it('Puts the correct colSpan on a loading meta-info cell', function() {
```

Comment:
I feel like all of these `colspan` tests would make more sense within the `NxTable.Body` describe block

Reply:
> I feel like all of these `colspan` tests would make more sense within the `NxTable.Body` describe block

I moved it here: cabfc93c988fb7f09a313275f3cb1dc19d9c7cd9